### DISCUSSION TITLE
error format 24 hour spiral cinnamon

### DISCUSSION THREAD

**Discussion by conocimientotransformador** [SpiralLinux/SpiralLinux-project#165] [Q&A] [answered]
I activate "use 24hour clock", but it still display 12h clock

**Comment by JohnnyCalavera** [comment, marked as answer]
Please take a look at the FAQ (https://github.com/SpiralLinux/SpiralLinux-project/wiki#faqs)
You need to disable custom format. (see also: https://github.com/SpiralLinux/SpiralLinux-project/discussions/29)

**Reply by geckolinux** [reply]
Thanks @JohnnyCalavera for helping to answer this question, that's the correct answer.
